src/Templates/OrchardCore.ProjectTemplates/content/OrchardCore.Templates.Cms.Module/Startup.cs

```diff
@@ -1,4 +1,6 @@
+#if (AddPart)
```

Comment:
```suggestion
```

Reply:
Yeah, with the new central imports indeed not, though I'm awaiting feedback on that.